scripts/safe_template_linter.py

```diff
@@ -217,13 +217,25 @@ class Rules(Enum):
         'mako-invalid-html-filter',
         'The expression is using an invalid filter in an HTML context.'
     )
+    mako_invalid_js_filter = (
+        'mako-invalid-js-filter',
+        'The expression is using an invalid filter in a JavaScript context.'
+    )
     mako_deprecated_display_name = (
         'mako-deprecated-display-name',
         'Replace deprecated display_name_with_default_escaped with display_name_with_default.'
     )
-    mako_invalid_js_filter = (
-        'mako-invalid-js-filter',
-        'The expression is using an invalid filter in a JavaScript context.'
+    mako_html_requires_text = (
+        'mako-html-requires-text',
+        'You must begin with Text() if you use HTML() during interpolation.'
+    )
+    mako_close_before_format = (
+        'mako-close-before-format',
+        'You must close any call to Text() or HTML() before calling format().'
```

Comment:
@nedbat: My rule more closely matches the current documentation (and the implementation of the rule).  I'd like to stick with it as-is and possibly adjust if and when people prove to be truly confused.  Is that ok with you?